bin/bugsnag-dsym-upload

```diff
@@ -13,18 +13,22 @@
 function print_usage() {
     echo "Usage: $0 [--symbol-maps DIR] dSYMS_PATH"
     echo
-    echo "-h, --help          Displays this message"
-    echo "-v, --verbose       Print verbose logging output during execution"
-    echo "--api-key API_KEY   The API key of the project the dSYM should be applied to"
-    echo "--symbol-maps DIR   Path to a directory of bitcode symbol maps. The"
-    echo "                    dSYM files will be restored with symbols prior to"
-    echo "                    upload. Requires dsymutil."
-    echo "--upload-server URL The server receiving dSYM files. Set this value if"
-    echo "                    using an on-premise Bugsnag installation"
-    echo "--project-root DIR  The root directory of the project. This will help to"
-    echo "                    group error reports by project"
-    echo "dSYMS_PATH          A directory or .zip file containing *.dSYM bundles to"
-    echo "                    upload"
+    echo "-h, --help                 Displays this message"
+    echo "-v, --verbose              Print verbose logging output during execution"
+    echo "-m, --allow-missing-dwarf  Throw warnings instead of errors when a dSYM with missing"
+    echo "                           DWARF data is found"
+    echo "-e, --ignore-empty-dsym    Throw warnings instead of errors when a *.dSYM file is found"
+    echo "                           rather than the expected *.dSYM directory"
```

Comment:
I'm not convinced the short options add anything here – it's not like it's an intuitive character nor an option that people are going to need to type in again and again. I suggest we remove it for consistency with the other non-standard options.

Reply:
I'd also suggest these should go as the last options in the usage text as they are a bit more niche than the others.

Reply:
allow changed for ignore, moved to bottom of options list: [`aa238a2` (#60)](https://github.com/bugsnag/bugsnag-dsym-upload/pull/60/commits/aa238a217d45bb9e48ae81022ff0d3f5623f89e9)